src/ui/QuerySuggestPreview/QuerySuggestPreview.ts

```diff
@@ -0,0 +1,93 @@
+import { ResultList } from '../ResultList/ResultList';
```

Comment:
Should we also extract the tests into a file `QuerySuggestPreviewTest.ts`?

Reply:
I will be a did not write any test yet